climakitae/derive_variables.py

```diff
@@ -3,14 +3,16 @@
 import numpy as np
 
 
-def compute_hdd_cdd(t2, standard_temp=65):
+def compute_hdd_cdd(t2, hdd_threshold=65, cdd_threshold=65):
     """Compute heating degree days (HDD) and cooling degree days (CDD)
 
     Parameters
     -----------
     t2: xr.DataArray
```

Comment:
Unrelated to the recent changes, it might be a good idea to add a check that ensures that it was indeed temperature that was passed into this function by checking the  metadata and throwing an error if not.